tests/models/test_project.py

```diff
@@ -63,9 +63,9 @@ def test_Project_validate_project_trace_refs():
     error = exc_info.value.errors()[0]
     assert (
         error["msg"]
-        == f'Value error, The reference "ref(trace_name)" on item "Chart - chart" does not point to an object.'
```

Comment:
@tgsoverly Not blocking merge- It could be helpful to do a fuzzy string match (maybe something like jaro-winkler >= 90) against other trace names in the product to make a suggestion for the trace that the user was trying to reference.